src/driver/mysql/MysqlDriver.ts

```diff
@@ -154,6 +154,19 @@ export class MysqlDriver implements Driver {
         }, this.options.extra || {});
 
         this.pool = this.mysql.createPool(options);
+        
+        return new Promise<void>((ok, fail) => {
```

Comment:
also please write a commend why are you doing it because its not really oblivious why do you need this code block